### DISCUSSION TITLE
M2 Mac Still doesn't start app automatically

### DISCUSSION THREAD

**Discussion by Yepitsmeposting** [MonitorControl/MonitorControl#1557] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Since I upgraded to M2, even with updated OSX and Monitor Control app, I cannot get the app to start automatically after a restart or power on. This is even though Monitor Control app is set to start on start up (see screenshots below). I also provided a shallow overview of my Mac Studio specs below. What can be done to resolve this? I really like the app but it is a pain to manually open it every time.
<img width="711" alt="Screenshot 2024-03-25 at 14 23 58" src="https://github.com/MonitorControl/MonitorControl/assets/152500173/59112504-4f69-40d0-a39a-d960413dff3e">
<img width="255" alt="Screenshot 2024-03-25 at 14 24 12" src="https://github.com/MonitorControl/MonitorControl/assets/152500173/123eb1b4-f0c1-4335-9e1e-829c30ba6f51">

### Steps to reproduce

I cannot get the app to start automatically after a restart or power on. This is even though Monitor Control app is set to start on start up

### Expected behavior

That I am after startup, able to change the brightness settings with my keyboard f1 and f2 keys without manually starting the app

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.2.1
- Mac model: M2 Mac Studio
- MonitorControl version:4.2.0
- Monitor(s): 2. BenQ GL2450H and Bell U2723QE
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by thisisrichardferreira** [comment]
I have an almost identical issue on an M2 machine: difference for me is that a volume with MonitorControl is mounted on login instead of actually launching the application. 

When I `Right Click > Show In Finder` it also mounts it instead of showing me the actual application. It's the only item that does that, all others show me the app within the application folder. Looks to be app specific.

@waydabber any thoughts?

**Comment by thisisrichardferreira** [comment]
Tagging @waydabber, @JoniVR, and @the0neyouseek as I see you as members of MonitorControl in [the contributors section](https://github.com/MonitorControl/MonitorControl/graphs/contributors): thanks for all your contributions, this tool has been a great help in my day-to-day.

I see more recent issues that were opened without much engagement, plus a PR from 2023 that's still awaiting review. Is this project still being maintained?

**Comment by waydabber** [comment]
Hi @thisisrichardferreira - not very actively. I plan to go through the issues and respond to them. But generally MonitorControl development has stalled, no new features or updates are planned right now. I accept PRs into the repo - the one that is not accepted is somewhat controversial so it's just not decided what to do with it.

I recommend trying [BetterDisplay](https://github.com/waydabber/BetterDisplay) which is in active development, it has all the features of MonitorControl for free (+ many more free and paid features). Most of the unimplemented (and now closed) feature requests for MonitorControl were implemented in BetterDisplay + all emerging issues are actively fixed.

Here is a comparison:

https://github.com/waydabber/BetterDisplay/wiki/List-of-free-and-Pro-features

**Comment by thisisrichardferreira** [comment]
Thanks @waydabber! That's helpful context for sure.

I liked that MonitorControl only did a few things and did them well, but happy to give BetterDisplay a go. Cheers, and thanks for all the great work!
